src/Prelude.purs

```diff
@@ -167,6 +130,18 @@ instance categoryFn :: Category (->) where
 class Functor f where
   map :: forall a b. (a -> b) -> f a -> f b
 
+infixl 4 map as <$>
+
+-- | `pam` is `map` with its arguments reversed. For example:
+-- |
+-- | ```purescript
+-- | [1, 2, 3] <#> \n -> n * n
+-- | ```
+pam :: forall f a b. Functor f => f a -> (a -> b) -> f b
```

Comment:
`mapBackwards`? This one is easier to remember, but I don't think we lose much by spelling it out.